common/src/main/java/com/revenuecat/purchases/common/Backend.kt

```diff
@@ -126,7 +134,7 @@ class Backend(
                 synchronized(this@Backend) {
                     callbacks.remove(cacheKey)
                 }?.forEach { (_, onError) ->
-                    onError(error)
+                    onError(error, false)
```

Comment:
can we add the parameter name here so we know what's being set to `false`?

Reply:
Hmm I think that's not allowed in Kotlin right? I get `Named arguments are not allowed for function types`. We could maybe pass this by passing an object instead of the parameters like this, but seems like overboard. I will just extract it to a constant before this, so at least we have the name clear here.